plugins/executor.js

```diff
@@ -38,7 +38,9 @@ const SCHEMA_START = Joi.object().keys({
         .label('API URI'),
     token: Joi.string().required()
         .label('Build JWT'),
-    enqueueTime: Joi.date().iso()
+    enqueueTime: Joi.date().iso(),
+    isPR: Joi.boolean().optional().default(false),
```

Comment:
I would say default `true` ? in case some caller forgets to set this, this will fall back on lower auth level